articles/channel.html

```diff
@@ -863,12 +863,12 @@ <h3><code>select-case</code> Control Flow Code Blocks</h3>
 	Later, a channel operation following a <code>case</code> keyword will be called a <code>case</code> operation.
 </li>
 <li>
-	In case of there are some non-blocking <code>case</code> operations,
-	Go runtime will <b>randomly select one of them to execute</b>,
+	If there is one or more non-blocking <code>case</code> operation,

```

Comment:
should be "... operations," here?

Reply:
I was on the fence, but if you think it should be plural, let's go with that!

https://english.stackexchange.com/questions/13284/which-is-correct-one-or-more-is-or-one-or-more-are#:~:text=In%20a%20sentence%20like%20one,is%20plural%20(more%20photos).&text=There%20is%20no%20general%20agreement,to%20be%20singular%20or%20plural.

Reply:
There's some conflicting advice out there and I could really go either way, so if you prefer plural let's go with that.